Demo/ViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class CollectionCell: UICollectionViewCell {
```

Comment:
Would be great if you'd put this cell in a different file, I think that is part of our best practices right?